test/retrieve_test.js

```diff
@@ -8,27 +9,22 @@ describe('finding records', function(){
 
     //create tests
 
-    beforeEach(function(done){
-
-        this.timeout(15000);
-    setTimeout(done, 15000);
-
-
-    var myArt = new Article({
+    it('Establish a connection to database', function(done) {
+        //ES6 Promises
+        mongoose.Promise = global.Promise;
     
-        title: 'Test Article',
-        author: 'Jaewon Lee',
-        body: "Article for test case"
-
-    });
-
-    myArt.save().then(function(){
-
-        assert(myArt.isNew === false); 
-        done();
-
-
-    });
+        //mongoose
+        mongoose.connect('mongodb://localhost/secret-admirer');
+        let db = mongoose.connection;
+    
+        db.once('open', function() {
+          done();
+        });
+    
+        db.on('error', function() {
+          done(new Error("Failed to connect secret-admirer database"));
+        });
+      });
 
 it('Retrieve one record from database', function(done){
```

Comment:
I don't think this test case is necessary.
Given that you have test case for sign up.
If you agree, please get rid of this file